### ISSUE TITLE
Add dataLayer to individual job page

### ISSUE THREAD

**Issue by mhoad** [mhoad/oaf-analytics#5]
Anywhere on the individual job page (example here: https://oaf-production.torchboxapps.com/work-with-us/job-openings/g/?gh_jid=660121) if you could add the following code:

```html
<script>
  dataLayer = [{
    'pageTemplate': 'Individual Job Page',
    'jobTitle': '{{ JOB_TITLE }}',
    'jobCountry': '{{ JOB_COUNTRY }}',
    'locationType': '{{ LOCATION_TYPE }}',
    'careerLevel': '{{ CAREER_LEVEL }}',
    'oafTeam': '{{ TEAM_NAME }}',
    'event': 'sendJobDetails'
  }];
</script>
```

Those variables should just be interpolated as per the information we already have on the page as shown below:
![image](https://user-images.githubusercontent.com/133005/27350098-9bd67c9e-55b6-11e7-961d-911293d0d7a2.png)

**Comment by phil-mcminn** [comment]
Please bear in mind the placement of your dataLayer code based on the recommendations I've made in issue #2. Your snippet of code should be in the head of your page above your GTM container snippet rather than anywhere on the page.

**Comment by mhoad** [comment]
Once again this one is entirely my bad. I had two things in my mind when putting that together that:

1. It may not be possible to know that data ahead of time when rendering the head (some other developers I have worked with struggled with this in different CMS platforms) but if it is not a concern here than no problems.
2. In my mind at least I was using dataLayer.push hence why I figured where on the page didn't matter especially when I had an event tagged onto it as well. Looking at the code again I clearly am not, thanks for catching that!

**Comment by nimasmi** [comment]
Jobs can have more than one country, location type, career level and team. Do you want us to send a list of all of them, or just the first?

**Comment by mhoad** [comment]
I've just spoken with Andrew at OAF about getting this picked up again at some point but I think all of them is probably the correct option here for our use case.

**Comment by todd-dembrey** [comment]
Hi @mhoad 

I've implemented the data layer as you requested above.

This results in the following datalayer for https://oneacrefund.org/work-with-us/job-openings/g/?gh_jid=232837:

```javascript
dataLayer = [{
    "pageTemplate": "Individual Job Page", 
    "jobTitle": "HR and People Specialist", 
    "jobCountry": ["Ethiopia"], 
    "locationType": ["Field"], 
    "careerLevel": ["Field and Support Professionals"], 
    "oafTeam": ["Program"], 
    "event": "sendJobDetails"
}];
```

Todd

**Comment by mhoad** [comment]
Hey @todd-dembrey sorry about the delay in getting back to you on this but I am not actually seeing this data appear in the dataLayer variable at all. Nor am I seeing the `sendJobDetails` event firing.  Just wondering if I am missing something obvious on my end potentially?

**Comment by todd-dembrey** [comment]
Hi @mhoad This isn't deployed to the live site yet, but is available on the staging site, I apologies, i wasn't that clear above. Do you have access to staging? I updated the main ticket with this information, but you may not have access to that. Let me know and i can give you more information.

**Comment by mhoad** [comment]
Yeah can you shoot me details to mark@afterwire.com.au when you have a
chance? Cheers
